docs/book/adapter/dbtable/credential-treatment.md

```diff
@@ -238,7 +260,9 @@ $sqlAlter = "ALTER TABLE [users] "
     . "AFTER [password]";
 ```
 
-Salts should be created *for each user* using a cryptographically sound pseudo-random number generator (CSPRNG). PHP 7 provides an implementation via `random_bytes`:
+Salts should be created *for each user* using a cryptographically sound pseudo-random number generator (CSPRNG).
+PHP 7 provides an implementation via `random_bytes` (and 
+[`random_compat` for older supported versions of PHP](https://github.com/paragonie/random_compat)):
```

Comment:
Rephrase this to `(and the [random_compat package provides them for older, supported versions of PHP](...))` (fill in the URL where I put ellipses).

Reply:
Since this package targets ^5.6, those functions are already available. Should we mention compat?

Reply:
Oh, nvm. `random_bytes()` is ^7.0